src/controllers/artist.js

```diff
@@ -0,0 +1,94 @@
+const getDb = require('../services/db');
+
+exports.create = async (req, res) => {
+    const db = await getDb();
+    const { name, genre } = req.body;
+  
+    try {
+      await db.query(`INSERT INTO Artist (name, genre) VALUES (?, ?)`, [
+        name,
+        genre,
+      ]);
+  
+      res.sendStatus(201);
+    } catch (err) {
+        console.log(err);
+      res.sendStatus(500).json(err);
+    }
```

Comment:
The formatting seems bit off in this file compared to album.js, 4 spaces were used for indentation there but in here its a mix of 2 and 4. Whichever you use is fine really, but they do need to be consistent across the project
```suggestion
    } catch (err) {
        console.log(err);
        res.sendStatus(500).json(err);
    }
```